src/app/containers/create-movie/displayError.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const DisplayError = ({ error }) => {
```

Comment:
The component DisplayError should not be aware of the full error object if it only uses the message

Reply:
solved at [6c63366](https://github.com/OctaFloare/flitnext-2024/pull/12/commits/6c63366ce437096d199fa5b5aafd3880fbd91fc9)